components/FeaturedCarousel.tsx

```diff
@@ -0,0 +1,125 @@
+import { SectionHeader } from "@/components/ui/section-header"
+import Provident from "@/components/assets/Provident"
+import BrownLogo from "@/components/assets/BrownLogo"
+import { ChevronLeftIcon, ChevronRightIcon, UserIcon } from "lucide-react"
+import { ProfileCard } from "@/components/ProfileCard"
+
+const featuredCarouselData = [
+  {
+    title: "PROVIDENT",
+    category: "Public Health",
+    description:
+      "A web app to support the PROVIDENT research study looking to prevent drug-related deaths in neighborhoods across Rhode Island.",
+    image: Provident,
+  },
+  {
+    title: "PROJECT 2",
+    category: "Economics",
+    description: "Hello world.",
+    image: BrownLogo,
+  },
+]
+
+export const FeaturedCarousel: React.FC = () => {
+  const [currentProjectIndex, setCurrentProjectIndex] = useState(0)
+
+  const handlePrevProject = () => {
+    setCurrentProjectIndex((prevIndex) =>
+      prevIndex === 0 ? featuredCarouselData.length - 1 : prevIndex - 1
+    )
+  }
+
+  const handleNextProject = () => {
+    setCurrentProjectIndex((prevIndex) =>
+      prevIndex === featuredCarouselData.length - 1 ? 0 : prevIndex + 1
+    )
+  }
+
+  const ImageComponent = featuredCarouselData[currentProjectIndex].image;
+
+  return (
+    <section className="w-full relative mt-[90px] flex items-center justify-center">
+      {/* Chevron Left Button */}
+      <Button
+        variant="outline_neutral"
+        size="icon"
+        iconOnly={<ChevronLeftIcon className="h-8 w-8 text-neutral-500" />}
+        className="absolute left-12 top-1/2 transform -translate-y-1/2 w-[45px] h-[45px] rounded-full bg-white border border-neutral-300 shadow-lg z-10"
+        onClick={handlePrevProject}
+      />
+
+      {/* Chevron Right Button */}
+      <Button
+        variant="outline_neutral"
+        size="icon"
+        iconOnly={<ChevronRightIcon className="h-8 w-8 text-neutral-500" />}
+        className="absolute right-12 top-1/2 transform -translate-y-1/2 w-[45px] h-[45px] rounded-full bg-white border border-neutral-300 shadow-lg z-10"
+        onClick={handleNextProject}
+      />
+
+      <div className="w-full">
+        <SectionHeader title="Featured Projects" align="center" />
+
+        <div className="w-full h-[588px] bg-white mt-[60px] relative">
+          <div className="flex justify-between px-12 max-w-screen-xl mx-auto">
+            <div className="max-w-[440px]">
+              <Badge className="bg-secondary-500 text-black rounded-[50px] font-semibold text-xs">
+                {featuredCarouselData[currentProjectIndex].category}
+              </Badge>
+
+              <h3 className="font-semibold text-black text-[28px] mt-10">
+                {featuredCarouselData[currentProjectIndex].title}
+              </h3>
+
+              <ProfileCard
+                icon={<UserIcon className="w-6 h-6" />}
+                name="Brown School of Public Health"
+                organization="RI Department of Health"
```

Comment:
The org should be PPHC. I'll update this in the figma too

```suggestion
                organization="People, Place and Health Collective"
```